packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-legend-label/gux-form-field-legend-label.scss

```diff
@@ -3,6 +3,10 @@
 /* stylelint-disable-next-line selector-class-pattern */
 @mixin Style {
   .gux-form-field-legend-label {
+    ::slotted(label) {
```

Comment:
added because it was allowing wrapping which was breaking the offsetWidth logic when no label position is specified. Also breaks other languages, like Japanese.